packages/go_router/example/lib/auth_bloc/bloc/authentication_bloc.dart

```diff
@@ -0,0 +1,41 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:equatable/equatable.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import '../type/authentication_status_type.dart';
+
+part 'authentication_event.dart';
+part 'authentication_state.dart';
+
+/// A [Bloc] which manages users authentication .
```

Comment:
trivial nit: you have an extra space before the period

Reply:
solved. please check again.